src/compiler/watchUtilities.ts

```diff
@@ -634,6 +643,25 @@ export function isIgnoredFileFromWildCardWatching({
             builderProgram.getState().fileInfos.has(file) :
             !!find(program as readonly string[], rootFile => toPath(rootFile) === file);
     }
+
+    function isSupportedScriptKind() {
+        if (!getScriptKind) return false;

```

Comment:
Should this fall back to `getScriptKindFromFileName` or something? If it's not provided, it seems like this returning false would make `isIgnoredFileFromWildCardWatching` always return true where it wouldn't have before?

(Doing that locally, nothing changed, so, maybe I don't know what I'm talking about)

Reply:
File extensions are always checked in is supported source name so it should have worked there so this is fallback so extensions’s additional answers where file is treated as ts/js etc are handled